### PR TITLE
Update FlutterBluePlusPlugin.java

### PR DESCRIPTION
**Pull request by hiperioncn**
Fix issue: ANDROID scan permission popup appears twice

### CODE REVIEW

**Comment by chipweinberger** [comment]
1. have you tested it?
2. please remove commented out code

File: android/src/main/java/com/lib/flutter_blue_plus/FlutterBluePlusPlugin.java [hunk 1]
```diff
@@ -1556,24 +1557,14 @@ private void askPermission(List<String> permissionsNeeded, OperationOnPermission
...
-            operationsOnPermission.remove(lastEventId);
-            if (!granted) {
-                operation.op(false, perm);
-                return;
-            }
-            // recursively ask for next permission
-            askPermission(permissionsNeeded, operation);
-        });
-
+        //Fix issue: ANDROID scan permission popup appears twice
```
Comment (chipweinberger) [inline android/src/main/java/com/lib/flutter_blue_plus/FlutterBluePlusPlugin.java:1560]: there's no need for this comment

Comment (hiperioncn) [reply android/src/main/java/com/lib/flutter_blue_plus/FlutterBluePlusPlugin.java]: Already remove it

**Comment by hiperioncn** [comment]
Test video recording
[

https://github.com/user-attachments/assets/33e5a5c1-b6fa-4c5c-9bd8-f06a97dc6dec

]

**Comment by chipweinberger** [comment]
I've updated FBP with this fix: https://github.com/chipweinberger/flutter_blue_plus/commit/1264f68afc47db2c08fdd0a7d3eca3f9982de40b

**note**: you'll need to call startScan *first* in your code to ask for both SCAN & CONNECT permission. If you just call `FlutterBluePlus.systemDevices` we will only ask for the CONNECT permission.

**Comment by hiperioncn** [comment]
Thanks, I've already request both SCAN&CONNECT permission before scan devices.

**Comment by chipweinberger** [comment]
your code caused a new problem: https://github.com/chipweinberger/flutter_blue_plus/pull/1013

**Pull request closed without merging**